packages/runtime/src/getGraphQLWSOptions.ts

```diff
@@ -23,17 +23,17 @@ export function getGraphQLWSOptions<TContext extends Record<string, any>, E>(
     execute: (args) => (args as EnvelopedExecutionArgs).rootValue.execute(args),
     subscribe: (args) =>
       (args as EnvelopedExecutionArgs).rootValue.subscribe(args),
-    onSubscribe: async (ctx, msg) => {
+    onSubscribe: async (ctx, _id, payload) => {
```

Comment:
This is a breaking change for v5 users. As we discussed on Slack with @enisdenjo, we will have a fallback logic here to keep supporting v5.

Reply:
Done.